web/src/main/resources/templates/fragments/marketplace-tables.html

```diff
@@ -0,0 +1,177 @@
+
+<!-- Jobs & Placements that are still pending approval -->
+<div th:fragment="pending-approval" id="pending-approval" xmlns:th="http://www.thymeleaf.org">
+    
+    <div class="row">
+        <div class="col-12 col-md-4 col-lg-4 col-xl-3"><h1 class="table-heading">Pending Approval</h1></div>
+        <div class="col-12 col-md-7 col-lg-8 col-xl-6"><input class="form-control" id="pendingApprovalFilter" type="text" name="keyword" placeholder="Filter Positions..." /></div>
+        <div class="col-md-1 col-xl-3"></div>
+    </div>
+    
+    <div class="row table-container">
+        <table class="table table-sm table-danger table-striped table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th>Type</th>
+                    <th>Provider</th>
+                    <th>Start Date</th>
+                    <th>End Date</th>
+                    <th>Description</th>
+                    <th class="text-center">Manage</th>
+                </tr>
+            </thead>
+            <tbody id="pendingPositionsData">
+                <tr th:each="pos : ${pendingPositions}">
+                    <th scope="row" th:text="${pos.id}"></th>
+                    <td th:text="${pos.type}">Job</td>
+                    <td th:text="${pos.partner.businessName}">Microsoft</td>
+                    <td th:text="${#dates.format(pos.startDate, 'dd/MM/yyyy')}">1/12/2022</td>
+                    <td th:text="${#dates.format(pos.endDate, 'dd/MM/yyyy')}">24/12/2022</td>
+                    <td th:text="${pos.description}">Just a one month job</td>
+                    <td style="display: flex;">
+                        <div class="mx-auto">
+                            <span class="material-symbols-rounded btn-approve">done</span>
+                            <span class="material-symbols-rounded btn-edit" title="Edit">edit</span>
```

Comment:
Button names not capitalized?

Reply:
this was picked up from the development branch already merged however i have fixed it